controllers/virtinkcluster_controller.go

```diff
@@ -138,10 +195,40 @@ func (r *VirtinkClusterReconciler) buildControlPlaneService(ctx context.Context,
 	}, nil
 }
 
+func buildInfrastructureClusterClient(ctx context.Context, c client.Client, secretRef *corev1.ObjectReference) (client.Client, error) {
```

Comment:
Why not make this func member of Reconciler?

Reply:
It will be used in both VirtinkCluster and VirtinkMachine controllers.